musicbrainz/index_test.go

```diff
@@ -207,3 +150,90 @@ func TestIndexArtists(t *testing.T) {
 		}
 	}
 }
+
+type testIndex struct {
+	db      *sql.DB
+	store   *meta.Store
+	artists []*Artist
+	tmpDir  string
+}
+
+func (t *testIndex) cleanup() {
+	if t.db != nil {
+		t.db.Close()
+	}
+	if t.tmpDir != "" {
+		os.RemoveAll(t.tmpDir)
+	}
+}
+
+func newTestIndex() (x *testIndex, err error) {
+	x = &testIndex{}
+	defer func() {
+		if err != nil {
+			x.cleanup()
+		}
+	}()
+
+	// load the test artists
+	f, err := os.Open("testdata/artists.json")
+	if err != nil {
+		return nil, err
+	}
+	defer f.Close()
+	dec := json.NewDecoder(f)
+	for {
+		var artist *Artist
+		err := dec.Decode(&artist)
+		if err == io.EOF {
+			break
+		} else if err != nil {
+			return nil, err
+		}
+		x.artists = append(x.artists, artist)
+	}
+
+	// store the artists in a test store
+	x.store = meta.NewStore(datastore.NewMapDatastore())
```

Comment:
Please consider explicit alias of `go-datastore` in import statement.

Reply:
`go-datastore` is not a valid reference in Go, what is wrong with `datastore` which is the actual package name?

Reply:
Same as `cid` above